var/spack/repos/builtin/packages/intel-xed/package.py

```diff
@@ -13,45 +13,45 @@ class IntelXed(Package):
 
     homepage = "https://intelxed.github.io/"
     git      = "https://github.com/intelxed/xed.git"
+    maintainers = ['mwkrentel']
 
-    # The version name and git commit hashes for the main xed repo and
-    # the mbuild resource.  Xed doesn't have official releases, only
-    # git commits.
-
-    version_list = [('2019.03.01',
-                     'b7231de4c808db821d64f4018d15412640c34113',
-                     '176544e1fb54b6bfb40f596111368981d287e951'),
-                    ('2018.02.14',
-                     '44d06033b69aef2c20ab01bfb518c52cd71bb537',
-                     'bb9123152a330c7fa1ff1a502950dc199c83e177')]
-
-    version('develop', branch='master')
-    resource(name='mbuild',
-             git='https://github.com/intelxed/mbuild.git',
-             branch='master', placement='mbuild',
-             when='@develop')
-
-    for (vers, xed_hash, mbuild_hash) in version_list:
-        version(vers, commit=xed_hash)
-        resource(name='mbuild',
-                 git='https://github.com/intelxed/mbuild.git',
-                 commit=mbuild_hash,
-                 when='@{0}'.format(vers))
+    mbuild_git = 'https://github.com/intelxed/mbuild.git'
+
+    # Current versions now have actual releases and tags.
+    version('master', branch='master')
+    version('12.0.1', tag='12.0.1')
+    version('11.2.0', tag='11.2.0')
+
+    # The old 2019.03.01 version (before there were tags).
+    version('10.2019.03', commit='b7231de4c808db821d64f4018d15412640c34113')
+
+    resource(name='mbuild', placement='mbuild', git=mbuild_git,
+             branch='master', when='@master')
+
+    resource(name='mbuild', placement='mbuild', git=mbuild_git,
+             commit='3e8eb33aada4153c21c4261b35e5f51f6e2019e8', when='@:999')
 
     variant('debug', default=False, description='Enable debug symbols')
+    variant('pic', default=False,
+            description='Compile with position independent code.')
 
-    # python module 'platform.linux_distribution' was removed in python 3.8
-    depends_on('python@2.7:3.7', type='build')
+    # The current mfile uses python3 by name.
+    depends_on('python+pythoncmd@3.4:', type='build')
```

Comment:
If it uses `python3` by name, it should not need `+pythoncmd`. That's only needed if it refers to it as `python`.